common/src/main/java/com/frostwire/search/one337x/One337xSearchPerformer.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml)
+ * Copyright (c) 2011-2018, FrostWire(R). All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.frostwire.search.one337x;
+
+import android.util.Log;
```

Comment:
please use the FrostWire logger instead of this one.

Reply:
`com.frostwire.util.Logger`

Reply:
One reason is consistency, another is that this will break the Desktop build.

Reply:
Surely, next time.
I am a beginner and its first time I have contributed to github project.
So I only used the basic one😅. 

AND again thanks for reviewing my pull requests ❤️